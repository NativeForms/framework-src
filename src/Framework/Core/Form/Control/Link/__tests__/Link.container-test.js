import React from 'react';
import { shallow } from 'enzyme';
import LinkContainer from '../Link.container';

describe('container <LinkContainer />', () => {
  it('container render correctly', () => {
    const wrapper = shallow(
      <LinkContainer />
    );
    expect(wrapper.dive()).toMatchSnapshot();
  });
});
