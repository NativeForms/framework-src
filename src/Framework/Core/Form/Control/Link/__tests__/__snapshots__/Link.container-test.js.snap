// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container <LinkContainer /> container render correctly 1`] = `
<Styled(Content)>
  <GridNB>
    <RowNB
      style={
        Object {
          "padding": 10,
        }
      }
    >
      <FormComponent
        messages={
          Object {
            "googleLabel": "Click to visit google!",
            "title": "Link",
          }
        }
        schema={
          Object {
            "attributes": Object {
              "formComponents": Array [
                Object {
                  "attributes": Object {
                    "label": Object {
                      "code": "googleLabel",
                    },
                    "route": "http://www.google.com",
                  },
                  "category": "control",
                  "type": "link",
                  "uid": "Link1",
                },
              ],
              "label": "Link Example",
            },
            "category": "group",
            "type": "section",
            "uid": "linkExample",
          }
        }
      />
    </RowNB>
    <RowNB
      style={true}
    >
      <SchemaViewer>
        export default {
          uid: 'linkExample',
          type: 'section',
          category: 'group',
          attributes: {
            label: 'Link Example',
            formComponents: [{
              uid: 'Link1',
              type: 'link',
              category: 'control',
              attributes: {
                label: {
                  code: 'googleLabel'
                },
                route: 'http://www.google.com'
              }
            }]
          }
        };
        
      </SchemaViewer>
    </RowNB>
  </GridNB>
</Styled(Content)>
`;
